.gitignore

```diff
@@ -155,6 +155,8 @@ objs
 /tools/json2json
 /tools/jsonstats
 /tools/minify
+/build/
```

Comment:
Can this just be one regex like /build.*/ or something?

Reply:
This aren't regex. Gitignore doest't have full regex support. These two are carefully tested to ensure that they don't ignore some files/folders that happen to have "build" in them. Often, when I build with cmake, I rename build to something like build-rem and regenerate project from scratch

Reply:
I believe as long as you prepend the / you only get top level (documentation and my experience so far agree on this point, anyway). `/build*/` would do the trick in that case.

Reply:
Yes, /build is the top level. In my case I actually had to suppress build folders within src tree and **/build*/ would ignore something that I didn't want. For top-level /build*/ should be ok. I can update the PR if you prefer

Reply:
Oh, I understand that one. If updating the PR isn't too much out of your way, it seems like the most expedient way forward :)

Reply:
done

Reply:
Thanks for your indulgence :)